.github/workflows/ci.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           if-no-files-found: error
           retention-days: 1
       - name: Test
-        run: ./mill -i java-class-name-tests.static.test
+        run: ./mill -i --no-filesystem-checker java-class-name-tests.static # TODO: get rid of --no-filesystem-checker
```

Comment:
This is to bypass the new restrictions on which task can do which IO operations in which directory, as per https://github.com/com-lihaoyi/mill/blame/9996ac655bea6653d7a815ecfb75f28bbf32ce2d/changelog.adoc#L211-L212
Rewriting the logic to not need it should be done as a follow-up.